content/actions/guides/about-continuous-integration.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: About continuous integration
+title: About continuous integration and continuous deployment
 intro: 'You can create custom continuous integration (CI) and continuous deployment (CD) workflows directly in your {% data variables.product.prodname_dotcom %} repository with {% data variables.product.prodname_actions %}.'
```

Comment:
Um relative and waist of time to readdatsa variables only true in drea. He has no rights to